### ISSUE TITLE
Has this project failed?

### ISSUE THREAD

**Issue by kcallis** [hacksalot/HackMyResume#229]
Just checking to see if this project is still being developed? I tried to pull from the registry and a no go there. I looked on AUR and it shows that there has been no updates since over a year ago. So I just wanted to see if this project has been sent to the **Island of Broken Toys** so don't end up wasting time.

**Comment by philipcunningham** [comment]
define what you mean by failed.

i've been using this project to build my resume on circleci for over a year without any issues.

otoh the project isn't being actively maintained.

**Comment by Suncatcher** [comment]
The [URL in description](https://fluentdesk.com/hackmyresume) is dead, does it mean that project is dead?

**Comment by kcallis** [comment]
Thank you @Suncatcher  for expanding on my original posting. I do understand @mcfilib that you are still making use of this project as well as myself, but the question is if the project will have any updates or changes. 

Most people who make use of OSS projects become some what complacent with updates and changes. But what happens when a component of the project changes. For instance, what if the schema changes significantly which although the use of an older version still works, the latest changes is no longer compatible with the old?

**Comment by Suncatcher** [comment]
> but the question is if the project will have any updates or changes

Indeed. I am also interested if it still worth investing time and effort in the project.

**Comment by mikelococo** [comment]
I was curious on the odds of future maintenance on the project so did some digging through history and found the following that might be useful to others. It seems pretty clear that HMR was either the technical seed or the community offering of FluentCV, which was intended to be some kind of career development related company. The last internet-facing vestiges of FluentCV folded up in 2017 or 2018. Given that FluentCV is gone, I find it unlikely that HMR will see future activity in this repo.

Interesting Sources:
- https://twitter.com/hackmyresume
- https://twitter.com/fluentcv
- As has been mentioned in other issues, fluentcv.com and the previous HMR website are both down and have been for quite some time: https://github.com/hacksalot/HackMyResume/issues/228
- https://news.ycombinator.com/item?id=10787608 An old hackernews post where hacksalot is active talking about some of the plans for the project at the time.
- https://github.com/karlitos/KissMyResume is not a fork, but an inspired project by someone who wanted something similar that they felt like they could maintain.

**Comment by karlitos** [comment]
Hi, I am the author of [KissMyResume](https://github.com/karlitos/KissMyResume) and I would like to explain, why did I decide to create a mew project, instead of contributing to this one.

I must admit, that I found the codebase of HackMyResume quite complex and overwhelming. 

- Class-based, object-oriented code which reminded me more on the classical lasagne-Java code, than something which I was used from JavaScript/Nodejs. Looking at the _generators_ - I could not even fully understand what was going on...

- Rendering dependant of local installation of Phantom.js, wkhtmltopdf, or WeasyPrint - something which caused many problems by the users

- I mentioned other reasons in the readme of my project

I wanted to create something which maybe wouldn't cover all the functionality of HackMyResume - (look at my TODO list) - but would serve as a simple and maintainable base

Therefore I decided to go with Google Puppeteer doing all the rendering of the templates,  use the resulting HTML markup as a starting point for further processing  and rely on as many 3rd party packages as possible. Regarding this was a one-man-show done in my free time ... I am quite pleased with the result.

I work currently on a GUI based on Electron, React and [jsonschema-form](https://github.com/rjsf-team/react-jsonschema-form) to make the resume generator accessible for wider audience. This will also saves me a lot of trouble providing hosting, taking care of personal user-data ...

I would like to point out, that HackMyResume provided some great inspiration and I definitely would'n say, that it _failed_ and I am very happy for the great work the HackMyResume developers did here 👍

**Comment by amruthpillai** [comment]
If you are looking for an alternative, might I suggest https://github.com/AmruthPillai/Reactive-Resume?

**Comment by drkarl** [comment]
> If you are looking for an alternative, might I suggest https://github.com/AmruthPillai/Reactive-Resume?

This doesn't look like an alternative, but a completely different project. HackMyResume was a CLI tool for a single user to generate their resumes, but Reactive-Resume is designed as a SaaS web application for many users, so it has a lot more complexity, with a front-end, database (Postgres), caching (Redis), object storage (Minio/S3), SMTP server to send emails to users, Google OAuth for authentication etc, it is WAY overkill for what a single users needs. I'd be happy with a command line tool without database or any of that complexity.

**Comment by xiaohanyu** [comment]
For anyone who is looking for an alternative CLI based resume/cv tools, you can check: https://yamlresume.dev/.

Features:

1. YAML based input
2. support [content](https://yamlresume.dev/docs/content/), [layouts](https://yamlresume.dev/docs/layouts) and [locale](https://yamlresume.dev/docs/locale) configurations
3. support multiple layouts/outputs, right now it supports LaTeX/PDF, HTML and Markdown
4. support multiple languages out of the box, for now it supports English, French, Norweigan, Dutch and Chinese
5. multiple templates, 3 in LaTeX and 2 in HTML
6. support [rich text in Markdown](https://yamlresume.dev/docs/content/rich-text)
7. support clang style error reporting, demo: https://asciinema.org/a/728098
8. dev mode (`npm run dev` to edit and watch resume builds), demo: http://localhost:3000/blog/dev-mode
9. docker image support: https://hub.docker.com/r/yamlresume/yamlresume
10. JSON Resume import: https://yamlresume.dev/docs/ecosystem/json2yamlresume

**Comment by awaisshah228** [comment]
For anyone looking for an active alternative — [Calcstack's CV Builder](https://calcstack.co/cv-editor) is actively maintained and has AI-powered features: content suggestions for bullet points and summaries, ATS compliance checking, cover letter generation, professional templates, and one-click PDF export. You can also bring your own API key (OpenAI, Anthropic, OpenRouter).
